### PR TITLE
chore(monorepo): put deps in root

### PR DESCRIPTION
**Pull request by shinGangan**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [x] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

It's about package management that I was talking about in this comment.

- https://github.com/nuxt/scripts/pull/86#issuecomment-2156449546

### CODE REVIEW

**Comment by shinGangan** [comment]
This PR is low priolity.
Please review when you have a time 🙏

File: docs/package.json [hunk 1]
```diff
@@ -23,8 +22,6 @@
     "@vueuse/core": "^10.10.1",
     "@vueuse/nuxt": "^10.10.1",
     "nuxt": "npm:nuxt-nightly@latest",
```
Comment (huang-julien) [inline docs/package.json:24]: @harlan-zw is it intentional to have `npm:nuxt-nightly@latest` ?

Comment (harlan-zw) [reply docs/package.json:24]: Just for testing purposes, happy to change it

**Comment by harlan-zw** [comment]
Thanks!

File: docs/package.json (hunk 1 above)
Comment (shinGangan) [reply docs/package.json:24]: OK, I'll put this change in a separate PR ;)

**Comment by shinGangan** [comment]
### Question

@harlan-zw 

Is it okay to manage `clinet/` in the same way?
I thought it would be better to separate it if there are separate npm releases etc., but if I'm mistaken, I'll manage it in the same way.

**Comment by harlan-zw** [comment]
They wouldn't have separate releases, I set it up like this based on Nuxt Fonts repo.

Happy to review anything changes you want to make though

**Comment by shinGangan** [comment]
Thanks for your comments. 💚
I will respond and create PR!

File: docs/package.json (hunk 1 above)
Comment (shinGangan) [reply docs/package.json:24]: I responded in this PR.

- nuxt/scripts#100

Comment (shinGangan) [reply docs/package.json:24]: Hi @huang-julien , I've changed this.
Please check it out when you have time. :)

**Pull request merged**